lib/hello_web/templates/page/index.xml.eex

```diff
@@ -0,0 +1,3 @@
+<user>
+    <name>Yaneth</name>
```

Comment:
@YanethM Please make sure your code editor is properly indenting your code. 1 Tab == 2 spaces